bigchaindb_driver/utils.py

```diff
@@ -0,0 +1,93 @@
+"""Set of utilities to support various functionalities of the driver.
+
+Attributes:
+    ops_map (dict): Mapping between operation strings and classes.
+        E.g.: The string ``'CREATE'`` is mapped to
+        :class:`~.CreateOperation`.
+
+"""
+from bigchaindb.common.transaction import Asset
+
+
+class CreateOperation:
+    """Class representing the ``'CREATE'`` transaction operation."""
+
+
+class TransferOperation:
+    """Class representing the ``'TRANSFER'`` transaction operation."""
+
+
+ops_map = {
```

Comment:
Use an Enum class instead, and just use values from that instead of strings for the two types of transactions. This should eventually land somewhere upstream.

Single dispatch is still possible with [`@valuedispatch`](http://lukasz.langa.pl/8/single-dispatch-generic-functions/#generic-history).